src/map/battle.c

```diff
@@ -4368,6 +4401,13 @@ struct Damage battle_attack_sc_bonus(struct Damage wd, struct block_list *src, s
 			ATK_ADD(wd.equipAtk, wd.equipAtk2, sc->data[SC_DRUMBATTLE]->val2);
 		if (sc->data[SC_MADNESSCANCEL])
 			ATK_ADD(wd.equipAtk, wd.equipAtk2, 100);
+		if (sc->data[SC_MAGICALBULLET]) {
```

Comment:
got compiling warning at this line:

battle.c: In function 'battle_attack_sc_bonus':
battle.c:4472: error: 'tstatus' undeclared (first use in this function)
battle.c:4472: error: (Each undeclared identifier is reported only once
battle.c:4472: error: for each function it appears in.)

Added: struct status_data *tstatus = status_get_status_data(target); and it compiled.